lib/cap-ec2/ec2-handler.rb

```diff
@@ -50,6 +50,10 @@ def stage
       Capistrano::Configuration.env.fetch(:stage).to_s
     end
 
+    def deployer
+      fetch(:deployer).to_s rescue ''
```

Comment:
Could we try 

```
fetch(:deployer, "")
```

Works on hashes, interesting to see if it works on this Cap class.

Reply:
You can provide a default value
```ruby
fetch(:deployer, "")
```
http://capistranorb.com/documentation/getting-started/configuration/